Parsers/Basic Math Parser.js

```diff
@@ -0,0 +1,17 @@
+/*
+activation_example: !add 5 10
+regex: !add (\d+) (\d+)
+flags: gmi
+order: 100
+stop_processing: true
+*/
+
+var matches = current.text.match(/!add (\d+) (\d+)/);
+if (matches) {
+    var num1 = parseInt(matches[1], 10);
```

Comment:
there are many more ways this can be implemented without being so restrictive on how it works